README.rst

```diff
@@ -268,6 +268,54 @@ There is another entry point for Android Studio - ``src/examples/facefilter/andr
 It should be used only for testing or as a template for starting your own project
 based on Drishti.
 
+Android Studio workarounds
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next factors add an instability to the Android Studio build:
```

Comment:
The following factors can all contribute to some instability in the Android Studio managed build: